### DISCUSSION TITLE
Open Source Summit & OSPOCon 2022 即将开始

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#95] [1️⃣ 新鲜事]
还有不到 12 个小时的 Open Source Summit & OSPOCon 2022 北美峰会即将开始，欢迎大家关注~

https://events.linuxfoundation.org/open-source-summit-north-america/

![4bf209bdb1451fb9882ab3d3e59864c](https://user-images.githubusercontent.com/15010826/174707528-cb12cc0a-049d-445c-99cf-6130d6ec9358.png)

其中，我们在本次峰会上与工业界会有一个 Panel，聊聊中国开源十年，以及国内 OSPO 发展的现状，同时也发布开源生态观测三件套的最后一个：OpenLeaderboard~

![d74ef7a65dea6287e31db89585e0ca1](https://user-images.githubusercontent.com/15010826/174707701-9dcc805b-79b1-4907-bc4d-0c7754e970d4.png)

至此，三大可视化套件就集齐了，面向全域的 **OpenLeaderboard**，面向组织的 **DataV 大屏**，以及面向仓库与开发者的 **Hypercrx**，分别是数据可视化在宏观、中观、微观上的实现~ 🎉🎉🎉

**Comment by TieWay59** [comment]
已经看过了所有的议程，等录播了。

**Reply by tyn1998** [reply]
666

**Comment by will-ww** [comment]
![0fbfe907c4677c61bfab384de5cde8d](https://user-images.githubusercontent.com/15010826/175162571-7364fe1f-8e29-4210-9a98-34c3871f9e1b.jpg)

![0e9d92d34c2e6b0aec2770595aee8ac](https://user-images.githubusercontent.com/15010826/175162631-9078e19e-82d9-4eac-9562-5b5459474d73.jpg)

感谢边 Sir 坚持到最后和大家互动~ 👏

**Comment by will-ww** [comment]
Ana 在Twitter 上转发的消息~ 👏

https://twitter.com/anajsana95/status/1539640213454131203
